src/styles/calendar.scss

```diff
@@ -0,0 +1,22 @@
+@import './variables.scss';
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: $gap-size;
+  width: calc($day-size * 7 + $gap-size * 6); // 7 колонок
```

Comment:
Ensure that `$day-size` is defined in your `variables.scss` file. Undefined variables will cause compilation errors.